src/en/proposals/ike709-robusthub.md

```diff
@@ -0,0 +1,96 @@
+- Official Discord/GitHub/Website/etc.
+- URL to pull News from (like the existing News tab in the SS14 launcher currently)
+- If the game is singleplayer, multiplayer, or both. 
+	- Multiplayer games will include the default/official Server Hub URL for the game
+	- Multiplayer games can decide whether or not to allow adding alternate unofficial hubs in the launcher (obviously SS14 would allow them, but Johnny GameDeveloper might want full control of the servers for his game)
+	- Exclusively-singleplayer games will point to the content bundle download URL 
+		- (Side note: the existing SS14 launcher still can't launch singleplayer games you've already downloaded like RobustSand)
+
+Adding a game to RobustHub should be as straightforward as creating a pull request and meeting whatever criteria is deemed sufficient by the RT project managers to be considered a separate game.
+
+The process of who will be permitted to update a game's hub entry and how will be fleshed out in a future design document dedicated to RobustHub pending this proposal's approval.
+
+## Launcher Changes
+ 
+ I couldn't find a way to articulate these launcher changes that doesn't make it sound a little crazy at first, so I advise reading all of this section before judging it.
+
+### Robust Launcher
+The SS14 launcher as it exists currently will no longer be maintained as a project. Instead the launcher will be stripped of all SS14 branding in favor of Robust Toolbox branding. The "Servers" tab will be nuked in favor of an interface similar to the BYOND pager, except with RobustHub games instead ([direct link](https://i.imgur.com/DSXdzyP.png)):
+
+<img src="https://i.imgur.com/DSXdzyP.png" />
+
+I don't expect it to look exactly the same; for example, we may have a separate page for singleplayer games. But players will be able to download, play, or browse the server list for any RobustHub game from Robust Launcher.
+
+#### Adding Alternate Server Hubs
+On the page for a specific game, there will be a button to configure the Server Hubs for that game (assuming the game allows alternate hubs in the RobustHub metadata).
+
+#### What About Other Game Hubs?
+Whether or not other Game Hubs should be addable to Robust Launcher is at the discretion of RT project managers. If they opt not to allow this functionality, other Game Hubs will ship their own launcher forks pointed at their hub instead of RobustHub.
+
+#### Optional: Steam Release
+I'm of the opinion that Robust Toolbox should have its own dedicated Steam page which ships Robust Launcher directly, to reduce confusion when people want to play other RT projects like OpenDream via Steam. I'm not saying we should do this *immediately*, but I think it's worth serious consideration if any not-SS14-adjacent projects are created in Robust Toolbox.
+
+We should also check Steam's policy for this sort of thing, as the SS14 launcher at this point would just be Robust Launcher in Game-specific Mode for SS14 (see below).
```

Comment:
IRT Steam's policy: Trackmania 2 (consisting of Canyon, Valley, Stadium, and Lagoon) and Shootmania collectively serve as existing precedent. None of these individual titles on Steam contain a full respective game, just the Maniaplanet launcher (which handles downloading and installing the actual game), with the title on Steam just quickstarting into a specific title via launch arg. Backing out of an individual title's main menu does not exit the game, but rather, puts you in Maniaplanet's home screen, regardless of whether or not it's the only Maniaplanet title you own.

We feel like game-specific mode isn't entirely necessary at all due to this precedent. Probably the only thing that'll be necessary is that the game merely boots into the given hub by default.

Reply:
The reason "Game-specific Mode" exists at all is because in prior discussions PJB outright refused to have a BYOND-style launcher that opened into a game list because it would lead to player confusion when they launch the Steam game "Space Station 14" and get a list of other games.

Reply:
I mean literally gmod.

Reply:
@PJB3005 Most gmod games don't have their own Steam store pages.